src/pages/markets/MarketsBanners.tsx

```diff
@@ -22,64 +19,45 @@ import { Output, OutputType } from '@/components/Output';
 import { useAppSelector } from '@/state/appTypes';
 import { getMarketIds } from '@/state/perpetualsSelectors';
 
-import { testFlags } from '@/lib/testFlags';
-
 export const MarketsBanners = () => {
   const stringGetter = useStringGetter();
-  const featureFlags = useAllStatsigGateValues();
-  const now = Date.now();
-  const trumpwinExpiration = new Date('2020-11-05T00:00:00Z').getTime();
   const { data: launchableMarkets } = useLaunchableMarkets();
   const marketIds = useAppSelector(getMarketIds, shallowEqual);
 
   return (
-    <>
-      {testFlags.pml && (
-        <$PmlBanner to={AppRoute.LaunchMarket}>
-          <img src="/affiliates-hedgie.png" alt="affiliates hedgie" tw="mt-1 h-5" />
-
-          <div tw="mr-auto flex flex-col">
-            <span tw="font-medium-medium">
-              {stringGetter({ key: STRING_KEYS.PERMISSIONLESS_LIVE })}
-            </span>
-            <span tw="text-color-text-0 font-base-book notTablet:text-nowrap">
-              {stringGetter({ key: STRING_KEYS.INSTANTLY_LAUNCH_BY_DEPOSITING })}
-            </span>
-
-            <$Details
-              withSeparators
-              layout="rowColumns"
-              items={[
-                {
-                  key: 'live',
-                  label: stringGetter({ key: STRING_KEYS.MARKETS }),
-                  value: <Output type={OutputType.CompactNumber} value={marketIds.length} />,
-                },
-                {
-                  key: 'launchable',
-                  label: stringGetter({ key: STRING_KEYS.LAUNCHABLE }),
-                  value: (
-                    <Output type={OutputType.CompactNumber} value={launchableMarkets.length} />
-                  ),
-                },
-              ]}
-            />
-          </div>
-
-          <$StarsOverlay />
-
-          <IconButton iconName={IconName.Arrow} />
-        </$PmlBanner>
-      )}
-
-      {featureFlags[StatsigFlags.ffShowPredictionMarketsUi] && now < trumpwinExpiration && (
-        <$MarketsPageBanner to={`${AppRoute.Trade}/${PREDICTION_MARKET.TRUMPWIN}`}>
-          <span>🇺🇸 {stringGetter({ key: STRING_KEYS.LEVERAGE_TRADE_US_ELECTION })}</span>
-          <$FlagOverlay />
-          <IconButton iconName={IconName.Arrow} />
-        </$MarketsPageBanner>
-      )}
-    </>
+    <$PmlBanner to={AppRoute.LaunchMarket}>
```

Comment:
why are we removing the feature flag check?

Reply:
ah will re-add